modules/mobile/app/serializers/mobile/v0/facility_eligibility_serializer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module Mobile
+  module V0
+    class FacilityEligibilitySerializer
+      include FastJsonapi::ObjectSerializer
+
+      set_type :FacilityEligibility
+      attributes :facility_id, :eligible, :reason
+
+      def initialize(facility_eligibilities, page_meta_data)
+        resource = facility_eligibilities.collect do |facility_eligibility|
+          FacilityEligibilityStruct.new(
+            facility_eligibility.facility_id,
+            facility_eligibility.facility_id,
```

Comment:
Setting both the id to the facility id as well as the adding the facility id as it's own field. since we're not using our own model, what goes in the id field is not intuitive.